applications/nrf_desktop/src/modules/dfu.c

```diff
@@ -94,7 +97,8 @@ const static char * const opt_descr[] = {
 	[DFU_OPT_DATA] = "data",
 	[DFU_OPT_SYNC] = "sync",
 	[DFU_OPT_REBOOT] = "reboot",
-	[DFU_OPT_FWINFO] = "fwinfo"
+	[DFU_OPT_FWINFO] = "fwinfo",
+	[DFU_OPT_BOOTLOADER_VAR] = "bootloader_var"
```

Comment:
please use:
```
[DFU_OPT_VARIANT] = OPT_DESCR_MODULE_VARIANT,
```

There is no reason to introduce new name.

Reply:
That could give us more flexibility while adding bootloader-specific functionalities. On the other hand it would require bigger changes on the Python side (from this point we will need to refer to the module using either `dfu/mcuboot` or `dfu/b0`)

Reply:
Ok we can introduce it. However there is a problem that right now that we cannot get variant name without knowing variant name (to fetch data you need to use command dfu/'variant_name'). I sugest to change current solution so that calling 
DEVICE config _module_ 
would return module variant.